apps/bfd-data-npi/src/main/java/gov/cms/bfd/data/npi/utility/FileNameCalculation.java

```diff
@@ -0,0 +1,60 @@
+package gov.cms.bfd.data.npi.utility;
+
+import java.util.Calendar;
+import java.util.HashMap;
+import java.util.Map;
+
+/** Extracts a file name. */
+public class FileNameCalculation {
+  /** Base url for the nppes download. */
+  private static final String BASE_URL = "https://download.cms.gov/nppes/NPPES_Data_Dissemination_";
+
+  /**
+   * Extracts a file name.
+   *
+   * @param getMonthBefore gets the month before
+   * @return a file name string
+   */
+  public static String getFileName(boolean getMonthBefore) {
+    Map<Integer, String> months =
+        new HashMap<Integer, String>() {
+          {
+            put(0, "January");
+            put(1, "February");
+            put(2, "March");
+            put(3, "April");
+            put(4, "May");
+            put(5, "June");
+            put(6, "July");
+            put(7, "August");
+            put(8, "September");
+            put(9, "October");
+            put(10, "November");
+            put(11, "December");
+          }
+        };
```

Comment:
This might be better as an immutable/final static constant.  It could be a List instead of a Map too since the keys are just ints from [0,11].